federation/source-config-handlers/src/main/java/org/codice/ddf/admin/sources/impl/test/ValidUrlTestMethod.java

```diff
@@ -17,32 +17,31 @@
 import static org.codice.ddf.admin.api.config.sources.SourceConfiguration.SOURCE_HOSTNAME;
 import static org.codice.ddf.admin.commons.requests.RequestUtils.CANNOT_CONNECT;
 import static org.codice.ddf.admin.commons.requests.RequestUtils.CONNECTED;
-import static org.codice.ddf.admin.commons.requests.RequestUtils.endpointIsReachable;
-import static org.codice.ddf.admin.commons.requests.RequestUtils.getRequestSubtypeDescriptions;
 
 import java.util.List;
 import java.util.Map;
 
 import org.codice.ddf.admin.api.config.sources.SourceConfiguration;
 import org.codice.ddf.admin.api.handler.method.TestMethod;
 import org.codice.ddf.admin.api.handler.report.Report;
+import org.codice.ddf.admin.commons.requests.RequestUtils;
 
 import com.google.common.collect.ImmutableList;
 
 // TODO: tbatie - 2/2/17 - (Ticket) This class should eventually be removed once the frontend is capable of handle errors messages from a probe report
 public class ValidUrlTestMethod extends TestMethod<SourceConfiguration> {
 
+    private static final RequestUtils REQUEST_UTILS = new RequestUtils();
+
     private static final String DESCRIPTION = "Attempts to connect to a given hostname and port";
 
     public static final String VALID_URL_TEST_ID = "valid-url";
 
     private static final List<String> REQUIRED_FIELDS = ImmutableList.of(SOURCE_HOSTNAME, PORT);
-
-    private static final Map<String, String> SUCCESS_TYPES =
+    private static final Map<String, String> SUCCESS_TYPES = REQUEST_UTILS.
```

Comment:
However in this case, I can see the advantage of it for these static finals. :man_shrugging: